annotation/tests/test_annotations.py

```diff
@@ -235,3 +274,188 @@ def test_create_manifest_json(
             annotated_doc.tenant,
             moto_s3,
         )
+
+
+def test_construct_annotated_doc(
+    annotated_doc: AnnotatedDoc,
+    annotation_doc_for_save: DocForSaveSchema,
+):
+    doc = annotation_doc_for_save
+    latest_doc = annotated_doc
+    is_latest = False
+    new_equal_latest = False
```

Comment:
you use them only once, lets remove unnecessary vars. for the clarity you should use kw args in the function you utilize these vars